lib/configsetup.inc.php

```diff
@@ -1629,6 +1629,18 @@
             'name' => 'protect[localhost_index]',
             'value' => $config['protect']['localhost_index'],
         ],
+        'protect_index_redirect' => [
+            'view' => 'advanced',
+            'type' => 'select',
+            'options' => [
+                'login' => 'Login',
+                'gallery.php' => 'Standalone Gallery',
+                'slideshow' => 'Standalone Slideshow',
+            ],
+            'placeholder' => $defaultConfig['protect']['index_redirect'],
```

Comment:
@andi34 do you still need the placeholder here, or is the first option always the default?

Reply:
Not sure to be true, used the language section as base and adjusted it